service/src/main/java/uk/gov/hmcts/reform/fpl/model/order/generated/ExclusionClause.java

```diff
@@ -7,5 +7,5 @@
 @Builder
 public class ExclusionClause {
     private final String exclusionClauseNeeded;
-    private final String exclusionClause;
+    private final String exclusionClauseText;
```

Comment:
this now does not match ccd def, 3 options
1. changed in fpl-ccd-configuration/ccd-definition/ComplexTypes/CareSupervision/1_exclusionClause.json, but it might require data migration (needs to check if filed has been already used in prod)
2. give a old name in @JsonProperty("exclusionClause") or JsonGetter
3. do not change this one at all

Reply:
Added @JsonProperty("exclusionClause)